core-sitemaps.php

```diff
@@ -18,6 +18,7 @@
  */
 
 const CORE_SITEMAPS_POSTS_PER_PAGE = 2000;
+const CORE_SITEMAPS_INDEX_MAX      = 50000;
```

Comment:
As this is also the maximum urls in any sitemap, would there be merit in reflecting that in the name?